src/Hamiltonians/HubbardRealSpace.jl

```diff
@@ -141,15 +202,16 @@ end
 
 # struct ================================================================================ #
 """
-    HubbardRealSpace(address; geometry=PeriodicBoundaries(M,), t=ones(C), u=ones(C, C), v=zeros(C, D))
+    HubbardRealSpace(address; geometry=PeriodicBoundaries(M,), t=ones(C, D), u=ones(C, C), Δ=zeros(C, C), v=zeros(C, D))
 
 Hubbard model in real space. Supports single or multi-component Fock state
 addresses (with `C` components) and various (rectangular) lattice geometries
 in `D` dimensions.
 
 ```math
-  \\hat{H} = -\\sum_{\\langle i,j\\rangle,σ} t_σ a^†_{iσ} a_{jσ} +
-  \\frac{1}{2}\\sum_{i,σ} u_{σσ} n_{iσ} (n_{iσ} - 1) +
+  \\hat{H} = -\\sum_{\\langle i,j\\rangle,σ} t_{iσ} a^†_{iσ} a_{jσ} +
+  \\frac{1}{2}\\sum_{i,σ,σ'} u_{σσ'} n_{iσ} (n_{iσ'} - δ_{σ,σ'}) + 
+  \\sum_{⟨i,j⟩,σ,σ'} Δ_{iσ,jσ'} n_{iσ} n_{jσ} +
```

Comment:
From the formula it looks like  Δ can vary across the lattice (it depends on the site indices and the component indices), but the function only accepts a D by D matrix. So should this be the following instead?
```suggestion
  \\sum_{⟨i,j⟩,σ,σ'} V'_{σ,σ'} n_{iσ} n_{jσ} +
```